packages/web-pkg/src/composables/appDefaults/useAppDefaults.ts

```diff
@@ -33,24 +34,29 @@ type AppDefaultsResult = AppConfigResult &
 export function useAppDefaults(options: AppDefaultsOptions): AppDefaultsResult {
   const router = useRouter()
   const store = useStore()
+  const currentRoute = useRoute()
   const clientService = options.clientService || defaultClientService
 
-  const currentRoute = computed(() => {
-    return router.currentRoute
-  })
-
   const isPublicLinkContext = computed(() => {
-    return unref(currentRoute).params.contextRouteName === 'files-public-files'
+    return unref(currentRoute).query[contextRouteNameKey] === 'files-public-files'
```

Comment:
Did you see the `useRouteQuery` composable? 🤔

Reply:
😅